java-viewer/src/test/resources/viewer_result3.html

```diff
@@ -0,0 +1,234 @@
+  <script src="lib/jquery-3.2.1/jquery.min.js"></script>
+  <script src="lib/bootstrap-3.3.7/js/bootstrap.min.js"></script>
+
+  <link rel="stylesheet" type="text/css" href="lib/codemirror-5.27.2/codemirror.min.css">
+  <script src="lib/codemirror-5.27.2/codemirror.min.js"></script>
+  <script src="lib/codemirror-5.27.2/mode/clike/clike.min.js"></script>
+  <script src="lib/codemirror-5.27.2/addon/selection/active-line.min.js"></script>
+
+  <link rel="stylesheet" type="text/css" href="lib/viewer/css/style.css">
+  <script src="lib/viewer/js/viewer.js"></script>
+
+  <script src="lib/vis-4.20.0/vis.min.js"></script>
+</head>
+<body>
+  <div class="container-fluid">
+    <div class="col-md-3">
+      <div class="row">
+        <h1>Graphs</h1>
+        <ul class="nav nav-pills">
+          <li role="presentation" class="active"><a  href="#eg" data-toggle="tab" class="btn btn-default">Exploded Graph</a></li>
+          <li role="presentation"><a href="#cfg" data-toggle="tab" class="btn btn-default">Control Flow Graph</a></li>
+          <li role="presentation"><a href="#ast" data-toggle="tab" class="btn btn-default">Syntax Tree</a></li>
+        </ul>
+      </div>
+      <div class="row">
+        <h1>Java sample</h1>
+        <form action="/" method="post">
+          <div class="codeWrapper">
+            <textarea id="java-code" name="javaCode">class B{}</textarea>
+          </div>
+          <div>
+            <button type="submit" class="btn btn-primary"><span class="glyphicon glyphicon-cog"></span>   Generate</button>
+            <a href="/" class="btn btn-default" role="button"><span class="glyphicon glyphicon-repeat"></span>   Reset</a>
+          </div>
+        </form>
+        <h1>Control Flow Graph</h1>
+        <div class="codeWrapper">
+          <textarea id="cfg-code">$cfg</textarea>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-9">
+      <div class="panel panel-danger" id="errorBox">
+        <div class="panel-heading">
+          <h3 class=panel-title>Unable to find a method in first class.</h3>
+        </div>
+        <div class="panel-body">
+          <p>java.lang.NullPointerException: Unable to find a method in first class.<br/>
+  at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:228)<br/>
+  at org.sonar.java.viewer.Viewer.getValues(Viewer.java:117)<br/>
+  at org.sonar.java.viewer.Viewer.generate(Viewer.java:102)<br/>
+  at org.sonar.java.viewer.Viewer.generate(Viewer.java:96)<br/>
+  at org.sonar.java.viewer.Viewer.lambda$startWebServer$2(Viewer.java:85)<br/>
+  at spark.RouteImpl$1.handle(RouteImpl.java:61)<br/>
+  at spark.http.matching.Routes.execute(Routes.java:61)<br/>
+  at spark.http.matching.MatcherFilter.doFilter(MatcherFilter.java:126)<br/>
+  at spark.embeddedserver.jetty.JettyHandler.doHandle(JettyHandler.java:50)<br/>
+  at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:189)<br/>
```

Comment:
As mentioned above, this is too fragile. Actually I had the test failing because of different path taken in jetty internals.